example/users/migrations/0001_initial.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
```

Comment:
Had to create initial migration for django 2.2. I added a comment in the PR that was a copy/paste of the line in the django 2.2 docs that gave me the hint for this fix.